news/PR858.dev

```diff
@@ -0,0 +1 @@
+bump Vagrant box to be based off Fedora 31
```

Comment:
```suggestion
Bump Vagrant box to Fedora 31
```

Reply:
Just a small suggestion, this will be in changelog later, so it's better if it's start with capital letter.

Reply:
Done.